src/components/Layout/Layout.jsx

```diff
@@ -0,0 +1,13 @@
+import Navbar from "../Navbar/NavBar.jsx"
+import Wrapper from "../Wrapper/Wrapper"
+
+const Layout = ({ children }) => {
+    return (
+        <>
+        <Navbar />
+        <Wrapper>{children}</Wrapper>
+        </>
```

Comment:
I think we can use an extra tab to indent this
```suggestion
        <>
            <Navbar />
            <Wrapper>{children}</Wrapper>
        </>
```

just a minor thing 😆